dash_app/dashboard.py

```diff
@@ -115,25 +175,27 @@ def init(server):
 
     region_df = region_table(per_timeseries_stats, ds)
 
-    df = per_timeseries_stats.stats.reset_index()
-    pivottable_data = [df.columns.tolist()] + df.values.tolist()
-
     dash_app.layout = html.Div(
         children=[
             html.H1(children="CAN Data Pipeline Dashboard"),
             html.P(commit_str),
             html.H2("Time series pivot table"),
+            html.P("Preset views:"),
+            *[
+                html.Button(preset.description, id=preset.btn_id)
+                for preset in TimeSeriesPivotTablePreset
+            ],
             dcc.Markdown(
                 "Drag attributes to explore information about time series in "
                 "this dataset. See an animated demo in the [Dash Pivottable docs]("
                 "https://github.com/plotly/dash-pivottable#readme)."
             ),
-            dash_pivottable.PivotTable(
-                id="pivot_table",
-                data=pivottable_data,
-                rows=[CommonFields.AGGREGATE_LEVEL],
-                cols=[timeseries_stats.FIELD_GROUP, timeseries_stats.DISTRIBUTION],
-            ),
+            # PivotTable `rows` and `cols` properties can not be modified as dash the Output of a
+            # dash callback, see
```

Comment:
Can't quite parse this comment ("cannot be modified as dash the Output of a dash callback" ?